nautobot_floor_plan/static/nautobot_floor_plan/js/label_utils.js

```diff
@@ -0,0 +1,145 @@
+function generateLabels(start, end, step, labelType, incrementLetter) {
+    let labels = [];
+    
+    if (labelType === "numbers") {
+        // Numeric labels
+        let startNum = parseInt(start, 10);
+        let endNum = parseInt(end, 10);
+        for (let i = startNum; i <= endNum; i += step) {
+            labels.push(i.toString().padStart(2, '0')); // Pad with leading zeros if necessary
+        }
+    } else if (labelType === "letters") {
+        // Letter labels
+        let currentCharCode = start.charCodeAt(0);
+        let endCharCode = end.charCodeAt(0);
+        while (currentCharCode <= endCharCode) {
+            labels.push(String.fromCharCode(currentCharCode));
+            currentCharCode += step;
```

Comment:
Any sort of special logic needed here (and in other cases) if e.g. I have `start=1, end=500`? or `step=1000`?